src/services/statistics/index.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const ua = require('universal-analytics');
+const hooks = require('./hooks');
+
+class Service {
+
+    constructor(options) {
+        this.options = options || {};
+        this.docs = {};
+
+    }
+
+    create(data, params) {
```

Comment:
we shoudl support batching later .. see oldags masterthesis for inout on this

Reply:
we have one request per page view only. because we want to measure offline duration time later batching will not be implemented